docs/data/material/components/popper/ScrollPlayground.js

```diff
@@ -71,9 +65,7 @@ const Popper = styled(MuiPopper, {
     },
   },
   '&[data-popper-placement*="left"]': {
-    '& > div': {
-      marginRight: arrow ? 2 : 0,
-    },
+    '& > div': {},
```

Comment:
nit: we can remove the definition if it is empty

Reply:
Removed.